doc/examples/jupyter_notebook/Makefile

```diff
@@ -1,12 +1,11 @@
 build:
 	docker build -t pachyderm_jupyter .
 
-prep: build
+prep:
 	pachctl version
-	./citibike_data/load.sh
-	./weather_data/load.sh
+	cd citibike_data && ./load.sh
+	cd weather_data && ./load.sh
 	pachctl create-pipeline -f ./sales_data/pipeline.json
-	pachctl create-job -f jupyter.json
```

Comment:
I agree - let's not automate this. It's unclear what caused your port forwarding issue but the succinct bug report for the invalid service is helpful